components/sysutils/cronie/Makefile

```diff
@@ -0,0 +1,63 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Adam Stevko
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		cronie
+COMPONENT_VERSION=	1.5.0
+COMPONENT_SUMMARY=	Cronie contains the standard UNIX daemon crond that runs specified programs at scheduled times and related tools.
+COMPONENT_PROJECT_URL=	https://fedorahosted.org/cronie/wiki
+COMPONENT_FMRI=		system/crond
```

Comment:
If cronie is going to be system/crond, what's the illumos cron going to be?  Just part of SUNWcs?